requirements/README.rst

```diff
@@ -32,3 +39,38 @@ If you instead need to surgically *downgrade* a dependency:
 2. Run ``make compile-requirements``
 
 This is considerably safer than trying to manually edit the ``*.txt`` files, which can easily result in incompatible dependency versions.
+
+Inconsistent dependencies
+*************************
+
+You might be directed to this section if a consistent-dependencies PR check has failed.
```

Comment:
```suggestion
You might be directed to this section if a ``consistent-dependencies`` PR check has failed.
```

Reply:
Since that's not the check's actual name, I'd be reluctant to put it in a code block.

(Or maybe the check should be renamed to that? :-P)

Reply:
Pushed up a slight rewording.